man/fitNullModel.Rd

```diff
@@ -34,7 +34,7 @@
 
   \code{isNullModelSmall} returns TRUE if a null model is small; FALSE otherwise.
 
-  \code{fitNullModelFastScore} fits a null model that can be used for association testing with the fast approximation to the score standard error (SE). 
+  \code{fitNullModelFastScore} fits a null model that can be used for association testing with the fast approximation to the score standard error (SE).
```

Comment:
@mconomos is this a reasonable update to the docs?

Reply:
@amstilp - I'd say it slightly differently to address other families with other link functions (e.g. poisson regression).

\item{fitted.values}{The fitted values from the model. For mixed models, this is X*beta where X is the design matrix and beta is the vector of effect size estimates for the fixed effects. For non-mixed models, this is the inverse link function applied to X*beta (e.g. expit(X*beta) for logistic regression when \code{family = "binomial"}.}

Reply:
Thanks!